pkg/agent/containerd/containerd.go

```diff
@@ -110,6 +116,7 @@ func Run(ctx context.Context, cfg *config.Node) error {
 	return preloadImages(ctx, cfg)
```

Comment:
we should probably wrap this in a lease, see: https://github.com/containerd/containerd/blob/master/docs/garbage-collection.md#how-to-use-leases

Reply:
Done!